starter-core/src/test/groovy/io/micronaut/starter/feature/function/azure/AzureHttpFunctionSpec.groovy

```diff
@@ -58,4 +62,21 @@ class AzureHttpFunctionSpec extends BeanContextSpec  implements CommandOutputFix
         and: 'link to Azure Gradle Plugin'
         readme.contains("The application's build uses [Azure Functions Plugin for Gradle](https://plugins.gradle.org/plugin/com.microsoft.azure.azurefunctions).")
     }
+
+    void 'test gradle azure-function-http feature for language=#language and buildTool=#buildTool'() {
+        when:
+        String template = new BuildBuilder(beanContext, buildTool)
+                .features(['azure-function-http'])
```

Comment:
azure-function-http is invisible. We should test `azure-function` and `ApplicationType.DEFAULT` and  `azure-function` and `ApplicationType.FUNCTION`